app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityPresenter.kt

```diff
@@ -1,12 +1,18 @@
 package org.oppia.android.app.devoptions
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import androidx.databinding.DataBindingUtil
 import org.oppia.android.app.activity.ActivityScope
 import org.oppia.android.app.databinding.databinding.DeveloperOptionsActivityBinding
 import org.oppia.android.app.drawer.NavigationDrawerFragment
+import org.oppia.android.app.splash.SplashActivity
 import org.oppia.android.app.ui.R
 import javax.inject.Inject
+import kotlin.system.exitProcess
+
+/** Tag for displaying [AppRestartDialogFragment]. */
```

Comment:
Note: Update with the latest changes to the filename.
```suggestion
/** Tag for displaying [ForceDownloadDialogFragment]. */
```

Reply:
updated